test/gtest/ucp/ucp_test.h

```diff
@@ -333,10 +333,7 @@ class ucp_test : public ucp_test_base,
 
 class test_ucp_context : public ucp_test {
 public:
-    static void get_test_variants(std::vector<ucp_test_variant> &variants)
-    {
-        add_variant(variants, UCP_FEATURE_TAG | UCP_FEATURE_WAKEUP);
-    }
+    static void get_test_variants(std::vector<ucp_test_variant> &variants);
```

Comment:
can this whole class be in CC file?

Reply:
Nope, it is used both in `test_ucp_context.cc` and in `test_ucp_tls.cc`